modules/csharp/partials/release-notes/couchbase-mobile-csharp-release-note.3.2.0.adoc

```diff
@@ -1,29 +1,165 @@
 [#maint-3-2-0]
-== 3.2.0 Beta 1 -- March 2024
```

Comment:
you don't want to keep the beta 1 release notes?
Oh, I see, they were completely lacking in useful info!

Reply:
There was no beta 1 release for .net